packages/dropdowns/examples/field-variants.md

```diff
@@ -0,0 +1,211 @@
+const {
+  Toggle,
+  Field: FormField,
+  Label: FormLabel,
+  Hint: FormHint
+} = require('@zendeskgarden/react-forms/src');
+
+const StyledSpacer = styled.div`
+  margin-top: ${props => props.theme.space.xs};
+`;
+
+const StyledField = styled(Field)`
+  display: ${state.inline && 'flex'};
+  align-items: ${state.inline && 'center'};
+`;
+
+const StyledHint = styled(Hint)`
+  display: ${state.inline && 'inline'};
+  margin-${props => (props.theme.rtl ? 'left' : 'right')}:
+    ${props => state.inline && props.theme.space.sm};
+`;
+
+const StyledLabel = styled(Label)`
+  margin-${props => (props.theme.rtl ? 'left' : 'right')}:
+    ${props => state.inline && props.theme.space.sm};
+`;
+
+const StyledMessage = styled(Message)`
+  margin-${props => (props.theme.rtl ? 'right' : 'left')}:
+    ${props => state.inline && props.theme.space.sm};
+`;
+
+initialState = {
+  regularWeightLabel: false,
+  showHint: true,
+  isCompact: false,
+  isBare: false,
+  focusInset: false,
+  disabled: false,
+  placeholder: false,
+  message: true,
+  validation: 'none'
+};
+
+<Grid>
+  <Row alignItems="center">
+    <Col size={4}>
+      <Well isRecessed>
+        <FormField>
+          <Toggle
+            checked={state.regularWeightLabel}
+            onChange={e => setState({ regularWeightLabel: e.target.checked })}
+          >
+            <FormLabel>Regular weight label</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle checked={state.showHint} onChange={e => setState({ showHint: e.target.checked })}>
+            <FormLabel>Hint</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle
+            checked={state.isCompact}
+            onChange={e => setState({ isCompact: e.target.checked })}
+          >
+            <FormLabel>Compact</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle checked={state.isBare} onChange={e => setState({ isBare: e.target.checked })}>
+            <FormLabel>Bare</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle
+            checked={state.focusInset}
+            onChange={e => setState({ focusInset: e.target.checked })}
+          >
+            <FormLabel>Focus inset</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle checked={state.disabled} onChange={e => setState({ disabled: e.target.checked })}>
+            <FormLabel>Disabled</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle
+            checked={state.placeholder}
+            onChange={e => setState({ placeholder: e.target.checked })}
+          >
+            <FormLabel>Placeholder</FormLabel>
+          </Toggle>
```

Comment:
What are "Placeholder" and "Focus inset" toggles supposed to demonstrate? I toggled them on and off repeatedly but didn't notice a difference in the side by side example:

![xx2020-02-05 15 18 46](https://user-images.githubusercontent.com/1811365/73892153-f2f21880-482a-11ea-8c2c-7770e8000bd9.gif)

Reply:
There is no `placeholder` on a native select. Let's get input from @allisonacs on whether the API should expose a `placeholder` prop on any of these. If so, then the example should show that working. If not, then `placeholder` needs to be removed as a prop in the API.

@hzhu toggle "Focus inset" and then focus on a field to observe the affect.

Reply:
I've removed the placeholder example for now until we decide how we want to approach this consistency.